lib/import/mobility.js

```diff
@@ -0,0 +1,192 @@
+var ArgumentParser = require('argparse').ArgumentParser;
+var mongoose = require('mongoose');
+var csv = require('fast-csv');
+var fs = require('fs');
+
+var config = require('../../config');
+var Mobility = require('../../app/models/mobility');
+
+/**
+ * Convert date from EST to ISO format
+ *
+ * @param{string} date - asdf
+ * @return{date}.
+ */
+function prep_isodate(date) {
+  date = new Date(date);
+  date.setHours(date.getHours());
+  return date.toISOString();
+}
+
+/**
+ * Import mobility from CSV to Mongo.
+ *
+ * @param{array} file - file in directory to read and import
+ * @param{string} country_code - lowercase ISO 3166-1 alpha-2 country code
+ * https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+ * @param{bool}verbose - whether to log messages while importing.
+ * @return{promise} Fulfilled once all records are saved.
+ */
+function import_file(file, country_code, verbose) {
+  var stream = fs.createReadStream(file);
+  var count = 0;
+  var mobility_objects = [];
+  // TODO(mikefab): consider reject here
+  return new Promise(function(resolve) {
+    var csvStream = csv()
+        .on('data', function(data) {
+          if (verbose && count % 1000 === 0) {console.log(count);}
+          // Skip first line (column names) in each mobility file
+          if ((count > 0) && (data.length > 3)) {
+            var o = {};
+            o.date = prep_isodate(data[0].replace(/(\d{4})(\d{2})(\d{2})/,
+                                                  '$1-$2-$3'));
```

Comment:
This has some logic here, and some logic in `prep_isodate`, though this is the only place that calls `prep_isodate`. We should consolidate this logic into a function that converts from "YYYYMMDD" strings to ISO strings. Something like this:

```
function yyyymmdd_to_iso(yyyymmdd) {
  var match = yyyymmdd.match(/(\d{4})(\d{2})(\d{2})/);
  var date = new Date(match[1], match[2], match[3]);
  return date.toISOString();
}
```